bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxLocalAccessHandler.java

```diff
@@ -190,12 +190,20 @@ private void updateChannels(RawDataParser parser, InverterData inverterData) {
                 supportedChannels);
         updateChannel(SolaxBindingConstants.CHANNEL_BATTERY_TEMPERATURE, inverterData.getBatteryTemperature(),
                 SIUnits.CELSIUS, supportedChannels);
+        updateChannel(SolaxBindingConstants.CHANNEL_INVERTER_TEMPERATURE1, inverterData.getInverterTemperature1(),
+                SIUnits.CELSIUS, supportedChannels);
+        updateChannel(SolaxBindingConstants.CHANNEL_INVERTER_TEMPERATURE2, inverterData.getInverterTemperature2(),
+                SIUnits.CELSIUS, supportedChannels);
         updateChannel(SolaxBindingConstants.CHANNEL_BATTERY_STATE_OF_CHARGE, inverterData.getBatteryLevel(),
                 Units.PERCENT, supportedChannels);
         updateChannel(SolaxBindingConstants.CHANNEL_FEED_IN_POWER, inverterData.getFeedInPower(), Units.WATT,
                 supportedChannels);
         updateChannel(SolaxBindingConstants.CHANNEL_POWER_USAGE, inverterData.getPowerUsage(), Units.WATT,
                 supportedChannels);
+        updateChannel(SolaxBindingConstants.CHANNEL_INVERTER_WORKMODE_CODE, inverterData.getInverterWorkModeCode(),
+                Units.ONE, supportedChannels);
```

Comment:
Not sure if Units.ONE is the most proper one. Maybe a good question to the approving reviewer. For me it's OK.

Reply:
I guess this is fine, in general i would personally prefer tekst channels for these kind of states. As there is no conversion/calculation so there is hardly any benefit in numbers and with a sstring also (future) text codes would be possible. Otherwise see general comment about the two channels.

Reply:
The text channel is there. I guess that Henrik added the code for additional information. Maybe we can switch this one to advanced, i.e. to be used only for rules and certain specific scenarios (even though I don't see any value if the relation is 1:1 code - enum value)